### ISSUE TITLE
Add POH Incense burner RuneLite notification

### ISSUE THREAD

**Issue by eocri** [runelite/runelite#11229]
There is currently a really well made POH plugin that includes an Incense burner timer. The timer turns green when a burner is first lit, and then orange when the burner enters the random timer zone.

While this is helpful, it can be frustrating to have to look at the screen the entire time during hosting, since the most important thing is that the burners stay lit non-stop.

I would like the plugin to trigger a RuneLite notification when either of the burners enters the orange zone, so that I can direct my attention elsewhere while the burners are lit. Much like how trade notifications, stat reductions and similar events can push the information through to a RuneLite notification.

**Comment by MSAskew** [comment]
Would it make more sense to notify a few seconds before the burners reach orange given that they could go out anytime once the orange timer is showing?

**Comment by eocri** [comment]
> Would it make more sense to notify a few seconds before the burners reach orange given that they could go out anytime once the orange timer is showing?

That would make sense. Though, I am not sure if the orange zone indeed represents the random interval. Purely speaking from experience, I have never had the lighters burn out during the orange zone.

The burner timers are explained on the osrs wiki (https://oldschool.runescape.wiki/w/Marble_incense_burners).

If the orange zone represents the variable zone, then it would indeed make sense to alert the user a few seconds prior.

**Comment by AFurryCat** [comment]
Yes the green circle represents the guaranteed uptime and the orange circle is when the burner enters the point where it could go out at any time.

5 seconds enough warning to click back into the client?

**Comment by eocri** [comment]
5 seconds would be perfect.
